batch/batch/front_end/front_end.py

```diff
@@ -1157,18 +1156,18 @@ async def insert(tx):
 
         bp_cost_record = await tx.execute_and_fetchone(
             '''
-SELECT billing_projects.msec_mcpu, COALESCE(SUM(`usage` * rate), 0) AS cost
+SELECT COALESCE(SUM(`usage` * rate), 0) AS cost
 FROM billing_projects
-INNER JOIN aggregated_billing_project_resources
-  ON billing_projects.name = aggregated_billing_project_resources.billing_project
+INNER JOIN aggregated_billing_project_user_resources_by_date
+  ON billing_projects.name = aggregated_billing_project_user_resources_by_date.billing_project
```

Comment:
I need to fix all of these queries to aggregate by resource first before computing the cost.